### PR TITLE
bf(S3C-3376): Remove requirement for keys from client

### PR DESCRIPTION
**Pull request by tmacro**

### CODE REVIEW

File: libV2/client/index.js [hunk 1]
```diff
@@ -83,8 +83,10 @@ class UtapiClient {
             accessKeyId: config && config.accessKeyId,
             secretAccessKey: config && config.secretAccessKey,
         };
-        assert.notStrictEqual(this._credentials.accessKeyId, undefined, 'you must provide an accessKeyId');
-        assert.notStrictEqual(this._credentials.secretAccessKey, undefined, 'you must provide a secretAccessKey');
+
+        if (!this._credentials.accessKeyId || !this._credentials.secretAccessKey) {
+            this._logger.warn('Now access or secret key provided, client limited to push only');
```
Comment (miniscruff) [inline libV2/client/index.js]:
```suggestion
            this._logger.warn('No access or secret key provided, client limited to push only');
```
I think this is what you meant.

**Review by miniscruff — APPROVED** [review]

**Review by naren-rajendran — APPROVED** [review]

File: libV2/client/index.js (hunk 1 above)
Comment (tmacro) [reply libV2/client/index.js]: I just realized I typoed the commit msg about a typo fix.

Comment (miniscruff) [reply libV2/client/index.js]: haha

**Pull request merged**